frontend/src/scenes/onboarding/sdks/sdk-install-instructions/android.tsx

```diff
@@ -16,24 +16,25 @@ function AndroidSetupSnippet(): JSX.Element {
     const { currentTeam } = useValues(teamLogic)
 
     return (
-        <CodeSnippet language={Language.Java}>
-            {`public class SampleApp extends Application {
-    private static final String POSTHOG_API_KEY = "${currentTeam?.api_token}";
-    private static final String POSTHOG_HOST = "${window.location.origin}";
-
-    @Override
-    public void onCreate() {
-        // Create a PostHog client with the given context, API key and host
-        PostHog posthog = new PostHog.Builder(this, POSTHOG_API_KEY, POSTHOG_HOST)
-            .captureApplicationLifecycleEvents() // Record certain application events automatically!
-            .recordScreenViews() // Record screen views automatically!
```

Comment:
They are enabled by default, so it's just omitted, the docs mention them more in detail.
The simpler/shorter, the better for a first UX.